includes/class-sensei-frontend.php

```diff
@@ -1237,10 +1237,22 @@ public function sensei_course_start() {
 	 * @return bool True if successful.
 	 */
 	public function manually_enrol_learner( $user_id, $course_id ) {
-		$enrolment_manager = Sensei_Course_Enrolment_Manager::instance();
-		$manual_enrolment  = $enrolment_manager->get_manual_enrolment_provider();
+		$course_enrolment = Sensei_Course_Enrolment::get_course_instance( $course_id );
 
-		return $manual_enrolment && $manual_enrolment->enrol_learner( $user_id, $course_id );
+		// If user is removed, just restore instead.
```

Comment:
I think _"instead"_ is not needed here, right?
I think it makes it a little confusing because it seems it's happening instead of something else.